index.js

```diff
@@ -396,20 +400,53 @@ vorpal
       },
       function(passphrase, seriesCb){
         var delegate = args.name;
-        var transaction = arkjs.delegates.createVote(passphrase);
-        self.prompt({
-          type: 'confirm',
-          name: 'continue',
-          default: false,
-          message: 'Sending '+arkamount/100000000+'ARK '+(currency?'('+currency+args.amount+') ':'')+'to '+args.recipient+' now',
-        }, function(result){
-          if (result.continue) {
-            return seriesCb(null, transaction);
-          }
-          else {
-            return seriesCb("Aborted.")
-          }
-        });
+        if (args.name === undefined) {
```

Comment:
this makes nonsense to derive the publickey of the votes from the passphrase (ie you would need to ask passphrase for the delegate you vote for...) I would remove this condition and suppose there is always a name.

Reply:
ok i am wrong, you are getting existing vote and remove it.

I would then change this way:
- `account unvote` -> remove vote if any
- `account vote <name>` -> vote for `name` and remove vote beforehand if any.

Reply:
Okay. Just to clarify, you just want me to tweak the "new vote" process, to un-vote the previous if applicable. You also want me to create a separate "command" which will be to unvote, if any.

Could you confirm that there can only ever be 1 vote per account please, or whether I should cater for removing all votes just in case?

Thanks!

Reply:
Yes there can be only 1 vote per account (unlike lisk for instance where there can be 101 votes per account)

Reply:
Okay, I will only cater for 1 vote.
When unvoting before creating a new vote, do you think it's worth waiting for the transaction to confirm? I was thinking of a standard setInterval to check every 5 seconds to see if the unvote transaction has confirmed. Does that sound acceptable?
I'm almost done implementing your suggested changes